stress/common/src/main/java/alluxio/stress/fuse/FuseIOTaskResult.java

```diff
@@ -136,303 +184,37 @@ public void setEndMs(long endMs) {
   }
 
   /**
-   * @return the map of thread counts to results
+   * @return the list of errors
    */
-  public Map<Integer, ThreadCountResult> getThreadCountResults() {
-    return mThreadCountResults;
+  public List<String> getErrors() {
+    return mErrors;
```

Comment:
beter to return a immutable version of the errors, if you don't want to external caller to modify the returned errors
e.g., `Collections.unmodifiableList(mErrors)`